Wordle.Api/Wordle.Api/Models/Player.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Wordle.Api.Models;
+
+public class Player
+{
+    public int PlayerId { get; set; }
+    public string Name { get; set; } = null!;
+    public int GameCount { get; set; }
+    public double AverageAttempts { get; set; }
+}

```

Comment:
could add 
```public double AverageSecondsPerGame { get; set; }```
To store how many seconds a game takes

Reply:
Definitely, but it's extra credit and I'll work on that if I have time.